### PR TITLE
Add: read-art && pm2-gui

### PR DESCRIPTION
**Pull request by Tjatse**
Have any question please feel free to tell me.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -359,6 +359,7 @@ Inspired by the [awesome](https://github.com/sindresorhus/awesome) list thing.
 - [pretty-error](https://github.com/AriaMinaei/pretty-error) - Errors with less clutter.
 - [humanize](https://github.com/taijinlee/humanize) - Data formatter for human readability.
 - [node-read](https://github.com/bndr/node-read) - Extract readable content from any page.
+- [read-art](https://github.com/Tjatse/node-readability) - Scrape/Crawl article from any site automatically. Make any web page readable, no matter Chinese or English.
```
Comment (sindresorhus) [inline readme.md]: How is this different/better than `node-read`? (this should really be written in the `read-art` readme).

Comment (Tjatse) [inline readme.md]: @sindresorhus That's a good question and you're really a meticulous owner, that's nice :p
I've worked on `node-read` from the initial phase - to help improving the performance and making it more stabler, you can read from [node-read's PRs](https://github.com/bndr/node-read/pulls?q=is%3Apr+is%3Aclosed), and I mentioned that in the [read-art md](https://github.com/Tjatse/node-readability#bndrnode-read):

> bndr/node-read

>I've contributed on this for a while, but it's hard to communicate with Vadim(we are in a different  timezone), and we have very different ideas. So I decided to write it on my own.

But to be honest, it still can not be used in a production env, lack of:
- Fit such a wide variety of domains (different URLs)
- Read speed
- Memory performance
- A fair mechanism for judging content

I've reconstructed the readability from Arc90's, and made a hight improvement:
- Faster Than Any Readability Module
- High Performance - Less memory
- Automatic Read Title & Content
- Follow Redirects
- Automatic Decoding Content Encodings(Avoid Messy Codes, Especially Chinese)
- Gzip/Deflate Support
- Proxy Support
- Generate User-Agent
- Different outputs: JSON, TEXT and HTML.

More important is that I am using `read-art` in the production - Spider/Crawl title && content from millions sites (only count domains). And it's very stable, quick and efficient.
data
body

**Comment by sindresorhus** [comment]
Thanks for the writeup. Make sure to include something like that in your readme.

I'm going to pass on the pm2gui thing. Looks nice, but not really interested in web gui's.

**Comment by Tjatse** [comment]
That's totally understandable.
I've included what I mentioned above in the the [readme](https://github.com/Tjatse/node-readability/blob/master/README.md) since being created. Never mind, maybe I just need more time to make it better and gain stars from those who really need it, thanks for your time :p

I've seen your commit. Maybe you wanna keep `node-read`, but  it's wrong [here](https://github.com/sindresorhus/awesome-nodejs#humanize), am I right?
currently:
```
[read-art](https://github.com/Tjatse/node-readability) - Extract readable content from any page.
```

should be:
```
[node-read](https://github.com/bndr/node-read) - Extract readable content from any page.
```

Please gonna to put it right.
Any way, appreciate that.

**Comment by sindresorhus** [comment]
Nope, it's correct. I'm not interested in two nearly identical modules, and yours looks better.

**Comment by Tjatse** [comment]
Uh, I misunderstood that, thanks.

**Pull request closed without merging**